coderadar-ui/src/app/view/main-dashboard/main-dashboard.component.ts

```diff
@@ -67,14 +78,37 @@ export class MainDashboardComponent implements OnInit {
     });
   }
 
+  openAddToTeamDialog(project: Project): void {
+    const dialogRef = this.dialog.open(AddProjectToTeamDialogComponent, {
+      width: '300px',
+      data: {teams: this.teams, project: project}
+    });
+
+    dialogRef.afterClosed().subscribe(result => {
+      if (result !== undefined) {
+        for(let team of result.teams) { //TODO: Check if project is already in team
```

Comment:
Hier ist noch ein TODO, bitte bearbeiten.

Reply:
Ich würde dieses TODO im nächsten PR bearbeiten.

Reply:
Gibt es dazu schon ein Issue? Wenn nicht kann das TODO in einem Issue festgehalten und aus dem Code entfernt werden.

Reply:
Issue #387 erstellt.